lib/Settings/PersonalSettings.php

```diff
@@ -43,32 +44,49 @@ class PersonalSettings implements ISettings {
 	 * @var IUserSession
 	 */
 	private $userSession;
+	/**
+	 * @var string
+	 */
+	private $ncVersion;
```

Comment:
We can remove the annotation at this case.

```suggestion
	private string $ncVersion;
```